Karma-Khameleons/bangazon-website
# Sam payment type view

## Description
This adds the view/functionality for creating a new Payment Type. Currently the view is accessed by following the link in the "products" view, however eventually it should be accessed from the profile/account page.

## Number of Fixes
1


## Related Ticket(s)
#4 

## Problem to Solve
```When a customer goes to the “create payment type” page```
```Then they should be prompted with all of the necessary form inputs for creating a valid payment type```

## Changes
bang_app/urls.py updated to route PaymentType related views
PaymentType views created


## Expected Behavior
Logged in users should be able to create new payment types by navigating to ```/payment_type/```


## Steps to Test Solution
1. From the root of the Django project, run the following commands:
```
git fetch --all
git checkout origin sam-paymentTypeView
python manage.py test bang_app
python manage.py runserver 8000
``` 
1. In the browser, go to ```http://localhost:8000/login/```
1. login
1. As a logged in user, navigate to the ```/payment_type``` view and create a new payment type.
1. Check that the new payment type has been saved to the database


## Unit Testing

## Documentation
